### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by markv2311**
Changed ReadMe and will continue to add on where needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+* [Npm]([https://www.npmjs.com/get-npm](https://www.npmjs.com/get-npm)) - The package manager
+
+## Version
+Not released 
+
+## Contributing 
+
+Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
+
+## Authors
```
Comment (chriscamac) [inline README.md:44]: make this section entitled "contributors" and have a link to the packages contributors page and a short text something like "Thank you to the contributors on this project."

File: README.md [hunk 2]
```diff
@@ -1 +1,54 @@
...
+What these tests actually test is if the code gets executed will it do what it is supposed to do. The most important thing when testing is making sure that the **Statements** are 100% covered. 
+
+## Built With 
+* [Angular]([https://angular.io/](https://angular.io/)) - The web framework used 
+* [Npm]([https://www.npmjs.com/get-npm](https://www.npmjs.com/get-npm)) - The package manager
+
+## Version
+Not released 
+
+## Contributing
```
Comment (chriscamac) [inline README.md:40]: we should remove this section, as it is talking about how to contribute and actually pointing to a file in someone else's repo.  we just aren't ready to discuss this yet and figure out a policy for this.  however you should put in an issue to bring it up and then add this back when we are ready.

**Review by chriscamac — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (markv2311) [reply README.md:40]: I deleted the contributing and took out the authors. Making a single contributors title.

File: README.md (hunk 1 above)
Comment (markv2311) [reply README.md:44]: I took out the authors and made it a single contributors page to just say thanks to the contributors

**Pull request closed without merging**